src/pages/Esg/Esg.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Pagination, Keyboard, Mousewheel } from 'swiper';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import './Esg.css';
+
+export default function Msg() {
+  const [slidesData, setSlidesData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/esgdata/esgData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setSlidesData(data);
+      });
+  }, []);
+
+  return (
+    <SwiperContainer>
+      <Swiper
+        direction="vertical"
+        pagination={{
+          clickable: true,
+        }}
+        modules={[Pagination, Keyboard, Mousewheel]}
+        mousewheel={true}
+        keyboard={true}
+      >
+        {slidesData.map(imgData => (
```

Comment:
```suggestion
        {slidesData.map(({id, video, imgUrl, mtitle, title}) => (
```
구조분해 할당해서 사용하는게 추후 유지보수에 좋습니다~

Reply:
map 함수 안에서도 구조분해를 하면 더 유지보수에 효과 적이군요!! 소중한 리뷰 감사합니다.